importer/helpers/dagbuilder.go

```diff
@@ -237,3 +345,104 @@ func (db *DagBuilderHelper) Maxlinks() int {
 func (db *DagBuilderHelper) Close() error {
 	return db.batch.Commit()
 }
+
+// AddNodeAndClose adds the last `ipld.Node` from the DAG and
+// closes the builder. It returns the same `node` passed as
+// argument.
+func (db *DagBuilderHelper) AddNodeAndClose(node ipld.Node) (ipld.Node, error) {
+	err := db.batch.Add(node)
+	if err != nil {
+		return nil, err
+	}
+
+	err = db.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	return node, nil
+}
+
+// FSNodeOverDag encapsulates an `ft.FSNode` that will be stored in a
```

Comment:
This might just be my general newness to Go, but I find the `ft` here confusing (I know I can look up the fact that it’s referring to the `unixfs` package at the top, but I keep having to do that when I’m new to files AND none of that shows up in the Godoc). What if we said:

> FSNodeOverDag encapsulates a UnixFS `fs.FSNode`

? Is that any better, or is it just confusing in a different way?

Reply:
It is *definetely* not you (see https://github.com/ipfs/go-ipfs/issues/5056).

> ? Is that any better, or is it just confusing in a different way?

I'd rather rename the package to its full name and have `unixfs.FSNode` (and yes, the `FS` in `FSNode` is also a confusing redundancy, that's what namespaces are for).